instrumentation/opentelemetry-instrumentation-dbapi/src/opentelemetry/instrumentation/dbapi/__init__.py

```diff
@@ -107,14 +107,15 @@ def wrap_connect(
     https://www.python.org/dev/peps/pep-0249/
 
     Args:
-        tracer: The :class:`opentelemetry.trace.Tracer` to use.
         connect_module: Module name where connect method is available.
         connect_method_name: The connect method name.
-        database_component: Database driver name or database name "JDBI",
-            "jdbc", "odbc", "postgreSQL".
+        database_system: An identifier for the database management system (DBMS)
+            product being used.
         database_type: The Database type. For any SQL database, "sql".
```

Comment:
should database_type be removed? i couldn't find it in the semantic conventions for databases

Reply:
Yes looks like there are no semantic convetions for it. It also doesn't seem to be used anywhere in the code so i'll remove it.